src/app/components/site/phy/FooterPhy.tsx

```diff
@@ -30,10 +30,13 @@ export const FooterPhy = () => (
                             <a href="https://www.cam.ac.uk/" target="_blank" rel="noopener" className="mt-2 mb-3">
                                 <img src="/assets/common/logos/university_of_cambridge.svg" alt='University of Cambridge website' className='footer-org-logo' />
                             </a>
-                            Funded by&nbsp;<ExternalLink href="https://www.gov.uk/government/organisations/department-for-education">
+                            Funded by {' '} <ExternalLink href="https://www.cam.ac.uk/">
+                            <strong>University of Cambridge</strong>.
+                            </ExternalLink>
```

Comment:
```suggestion
                            Funded by {' '} <ExternalLink href="https://www.cam.ac.uk/">
                                <strong>University of Cambridge</strong>
                            </ExternalLink>.
```
This is my mistake, the ESLint server wasn't configured properly after I updated it. It should (now) flag misaligned indents for you!